bundler/spec/runtime/with_unbundled_env_spec.rb

```diff
@@ -116,6 +106,17 @@ def run_bundler_script(env, script)
     end
   end
 
+  shared_examples_for "an unbundling helper" do
```

Comment:
This is what is left of the shared example that the other specs were moved out of.